mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule Razor.Mixfile do
   
   def project do
     [app: :razor,
-     version: "0.0.1",
+     version: "0.0.2",
```

Comment:
Let's keep this at `0.0.1` please, since nothing has really changes & nobody is dependent.